lib/getPostcssResult.js

```diff
@@ -39,40 +39,19 @@ module.exports = function (stylelint, options = {}) {
 
 	return getCode
 		.then((code) => {
+			if (stylelint._options.syntax) {
+				throw new Error(
+					'The syntax option has been removed. Install the appropriate syntax and use the customSyntax option instead',
```

Comment:
[suggestion] How about improving the message as below?

- Emphasize the option names with quotes.
- Add the CLI flag names.

```suggestion
					'The "syntax" (--syntax) option has been removed. Install the appropriate syntax and use the "customSyntax" (--custom-syntax) option instead',
```

Reply:
SGTM. We can revisit the message in https://github.com/stylelint/stylelint/issues/5301 to make it consistent with the other messaging.